tests_new/integration_tests/modules/dashboards/test_dashboard.py

```diff
@@ -0,0 +1,113 @@
+from assertpy import assert_that
+
+from integration_tests.modules.dashboards.queries import (
+    search_dashboards,
+    get_dashboard,
+    list_dashboard_shares,
+    get_author_session,
+    get_reader_session,
+)
+from integration_tests.modules.dashboards.mutations import (
+    update_dashboard,
+    delete_dashboard,
+    approve_dashboard_share,
+    reject_dashboard_share,
+)
+from integration_tests.modules.dashboards.conftest import create_dataall_dashboard
+from integration_tests.errors import GqlError
+
+UPDATED_DESC = 'new description'
+
+
+def test_get_author_session(client1, persistent_env1):
+    assert_that(get_author_session(client1, persistent_env1.environmentUri)).starts_with('https://')
+
+
+def test_get_author_session_unauthorized(client2, persistent_env1):
+    assert_that(get_author_session).raises(GqlError).when_called_with(client2, persistent_env1.environmentUri).contains(
+        'UnauthorizedOperation', 'CREATE_DASHBOARD'
+    )
+
+
+def test_get_dashboard(session_id, dashboard1):
+    assert_that(dashboard1.label).is_equal_to(session_id)
+
+
+def test_list_dashboards(client1, client2, session_id, dashboard1):
+    filter = {'term': session_id}
+    assert_that(search_dashboards(client1, filter).nodes).is_length(1)
+    assert_that(search_dashboards(client2, filter).nodes).is_length(0)
+
+
+def test_get_dashboard_unauthorized(client2, dashboard1):
+    assert_that(get_dashboard).raises(GqlError).when_called_with(client2, dashboard1.dashboardUri).contains(
+        'UnauthorizedOperation', 'GET_DASHBOARD'
+    )
+
```

Comment:
nit: add the dashboard uri to the assertions to make sure we are checking the correct resource (Applicable to other tests)

Reply:
done!